data/podcasts.json

```diff
@@ -573,15 +562,15 @@
     "broadcast": "mondays",
     "logo": "https://shoptalkshow.com/wp-content/themes/shoptalk2/images/shoptalk_logo.png"
 },{
-    "name": "TED Talks Daily",
-    "host": "TED",
-    "url": "https://www.ted.com/about/programs-initiatives/ted-talks/ted-talks-audio",
-    "category": "ideas, general, stories",
-    "description": "Every weekday, this feed brings you our latest talks in audio format. Hear thought-provoking ideas on every subject imaginable -- from Artificial Intelligence to Zoology, and everything in between -- given by the world's leading thinkers and doers.",
+    "name": "Hackable?",
+    "host": "Justin",
+    "url": "https://hackablepodcast.com/",
+    "category": "Security",
+    "description": "Curious about how cybercriminals get all of your information? 'Hackable?' gets into the mind of a hacker to see how easy it is to crack into Wi-Fi, cars, webcams, and more.",
     "languages": "English",
-    "updates": "daily",
-    "broadcast": "week days",
-    "logo": "https://pi.tedcdn.com/r/pb-assets.tedcdn.com/system/baubles/files/000/005/187/original/TED_Talks_Podcast_Thumbnail_Audio_600px.png?w=320"
```

Comment:
and here thank you.

Have an awesome hacktoberfest week 😄

Reply:
I believe I have added them back in. I'm still a bit confused how i ended up removing them, but thank you for checking on it and catching it!

Reply:
Awesome, Thank you so much.